uniffi/Cargo.toml

```diff
@@ -13,6 +13,11 @@ license = "MPL-2.0"
 edition = "2021"
 keywords = ["ffi", "bindgen"]
 
+[[bin]]
+name = "uniffi"
+path = "src/main.rs"
+required-features = ["cli"]
```

Comment:
This means we're shipping a binary again, which we only recently removed.
Is that required? Shouldn't it still be the consumer building the binary to run to avoid versionining issues?

Reply:
Yeah, that's a good point.  It was useful for testing, but I'll remove that commit from this PR.